src/mono/wasm/wasm.targets

```diff
@@ -24,7 +24,6 @@
   <Target Name="CheckEnv">
     <Error Condition="'$(TargetArchitecture)' != 'wasm'" Text="Expected TargetArchitecture==wasm, got '$(TargetArchitecture)'"/>
     <Error Condition="'$(TargetOS)' != 'Browser'" Text="Expected TargetOS==Browser, got '$(TargetOS)'"/>
-    <Error Condition="'$(EMSDK_PATH)' == ''" Text="The EMSDK_PATH environment variable should be set pointing to the emscripten SDK root dir."/>
```

Comment:
I think we should consider keeping this and having a way to disable the automatic provisioning, maybe with a separate environment variable

Reply:
Its set automatically to src/mono/wasm/emsdk.